tools/tracy/include/inviwo/tracy/tracy.h

```diff
@@ -0,0 +1,169 @@
+#pragma once
```

Comment:
might want to disable clang format here `// clang-format off`

Reply:
yes...